mantidproject/mantid
# 24437 rm line multi plot

The multiplotting now includes vertical lines, annotations and the ability to remove lines. 

**To test:**
Open the multi plotting interface (have to enable it first)
You will see some vertical lines and annotations
Click `remove line`
You should be able to remove everything excpet for the line and annotation marked as protected.



Fixes #24437. 



Does not need to be in the release notes. 
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
